doc/m4/FS22-Group-36-M4-Report.md

```diff
@@ -0,0 +1,97 @@
+
+Report for Milestone 3 of Sopra FS22, Group 36
+
+![cover (source B. Furrer)](../header/jass-image.png)
+
+**Members**
+
+Carmen Kirchdorfer (20-720-132)  
+Salome Wildermuth (10-289-544)  
+Beat Furrer, group leader (07-542-392)  
+Lucius Bachmann (11-060-274)  
+Moris Camporesi (19-764-349)
+
+\newpage
+\raggedright
+## 1.  Introduction to the game
+Our game can be played by two up to five people. The goal for every player is to gain as 
+many points as possible during the game. Every player plays for himself, 
+there are no teams. The game is played with a “Swiss Jass” card set of 36 cards with four suits.
+
+The game is divided into 9 matches (Runden) with changing number of distributed cards per player 
+(5,4,3,2,1,2,3,4,5), and for every match points are distributed.
+
+For the first match, every player gets 5 cards. 
+Then one player after the other must announce how many tricks ("Stich", according to https://en.wikipedia.org/wiki/Jass) 
+they will make in this match, 
+starting from the player right from the current dealer. 
+The last one announcing the number of his tricks for this match must announce several tricks such that the total 
+sum of announced tricks is not equal to the number of cards distributed per player in this match.
+
+
+When the trick announcing round is completed, the game starts. The player to the right of the current dealer starts. 
+The suits don’t matter in this game, just the rank, and there is no trump. The player who played the highest 
+rank in a match, i.e. wins the trick and has to deal the first card for the next trick.
+If there are 2 or more highest ranks in the trick, the next trick stacks on top of the current. In this case, 
+the player who started the current trick has to start the next one too. If the last trick was stacked, the players 
+who played the highest cards draw another card and continue the next trick with these. This may continue until 
+there is only one highest rank in the trick.
+
+When all the tricks in the match are played out, the players count the number of their tricks.
+
+
+Points counting rules:\
+If a player announced the number of tricks correctly, then he gets the number of announced tricks squared as 
+positive points. Otherwise, the player gets the difference as negative points.
+
+Then the cards are distributed for the next match. The dealer duty moves one player to the right, the number 
+of cards that is distributed follows the sequence [5,4,3,2,1,2,3,4,5].
+
+For the match with only 1 card there is a special rule. The players don’t see their own card but put their card 
+with the front revealed on their forehead, so only the other players can see the card, but not the player himself. 
+After the 9 matches, the player with the highest score wins.
+
+## 2. Game view
+The order of the screenshots corresponds to the flow of the game, if possible. Matches 4, 3, 2 and the last match 
+are not listed. The only difference to the five-card match is the number of cards.
```

Comment:
What do you mean with _Matches 4, 3, 2, are not listed._ ?

Reply:
I mean rounds...thx Sherlok! ;-)